server/belga/search_providers.py

```diff
@@ -481,7 +481,7 @@ def format_list_item(self, data):
 
         return {
             "type": self.get_type(data.get("assetType", "text")),
-            "mimetype": "application/superdesk.item.text",
+            "mimetype": f"application/superdesk.item.{data.get('assetType','text').lower()}",
```

Comment:
would be better to use that `get_type` helper so we're only using types we can handle